src/Compilers/CSharp/Portable/PublicAPI.Shipped.txt

```diff
@@ -161,9 +161,12 @@ Microsoft.CodeAnalysis.CSharp.Syntax.AccessorDeclarationSyntax.Body.get -> Micro
 Microsoft.CodeAnalysis.CSharp.Syntax.AccessorDeclarationSyntax.Keyword.get -> Microsoft.CodeAnalysis.SyntaxToken
 Microsoft.CodeAnalysis.CSharp.Syntax.AccessorDeclarationSyntax.Modifiers.get -> Microsoft.CodeAnalysis.SyntaxTokenList
 Microsoft.CodeAnalysis.CSharp.Syntax.AccessorDeclarationSyntax.SemicolonToken.get -> Microsoft.CodeAnalysis.SyntaxToken
+Microsoft.CodeAnalysis.CSharp.Syntax.AccessorDeclarationSyntax.ExpressionBody.get -> Microsoft.CodeAnalysis.CSharp.Syntax.ArrowExpressionClauseSyntax
```

Comment:
All of the changes in this file should actually be made to the `PublicAPI.Unshipped.txt` file instead, please. We'll move the contents when we "ship".